docs/reference/postgresql12_migration.md

```diff
@@ -0,0 +1,71 @@
+# Migrate from PostgreSQL 12
+
+!!! info "Important"
+    Percona Everest 1.6.0 has officially discontinued support for PostgreSQL 12. To ensure compatibility, security, and access to new features, it is highly recommended to migrate to PostgreSQL 13 or later.
+
+## Preparation before migration
+
+Before migrating from PostgreSQL 12, it's important to prepare thoroughly to ensure a smooth transition. Here are the key steps:
+{.power-number}
+
+1. **Backup your data** - Create a complete backup of your data before upgrading.
+
+2. **Check Compatibility** - Verify that your extensions and configurations are compatible with the new PostgreSQL version.
+
+3. **Install the new version** – Download and install the target PostgreSQL release.
+
+
+## Choose a migration option
+
+When upgrading from PostgreSQL 12, it is essential to choose a migration method that suits your environment. For a smooth transition, consider the following factors during migration:
+
+- Data size
+
+- Downtime tolerance
+
+- Infrastructure (Kubernetes, cloud, on-premise)
```

Comment:
Everest can run on Kubernetes (unless you meant on-premise Kubernetes, in which case we should rephrase this)

Reply:
Removed it.